src/PrestaShopBundle/Form/Admin/Improve/International/Currencies/CurrencyType.php

```diff
@@ -213,22 +244,48 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                             ]
                         ),
                     ]),
+                    /*
+                     * I added this constraint because if range is too big it causes an "out of range" error in Vue.
+                     * I chose maximum precision based on this
+                     * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Precision_range
```

Comment:
Well with 20 decimals I think we are quite safe 👍  😄 

Nice idea to use Mozilla reference